sentry/src/lib.rs

```diff
@@ -111,5 +111,26 @@
 //! * `with_test_support`: Enables the test support module.
 #![warn(missing_docs)]
 
+#[macro_use]
+extern crate sentry_core;
```

Comment:
Instead of the macro use, can you just use a qualified import to `sentry_core::sentry_debug`? That may not work due to the inner macro invocation, so you may need to qualify this with `$crate::`.

Reply:
nice! Still had to export that macro though, and also pull in `log` as another dependency. I would like to overhaul the complete logging at some point either way.